src/utils/particle/drawer.ts

```diff
@@ -174,12 +188,17 @@ export class Drawer {
         ctx.drawImage(img_obj, particle.position.x - radius, particle.position.y - radius, radius * 2, radius * 2 / ratio);
     }
 
-    public static subDrawShape(ctx: CanvasRenderingContext2D, startX: number, startY: number, sideLength: number, sideCountNumerator: number, sideCountDenominator: number) {
+    private static subDrawShape(ctx: CanvasRenderingContext2D,
+        startX: number,
+        startY: number,
+        sideLength: number,
+        sideCountNumerator: number,
+        sideCountDenominator: number): void {
         // By Programming Thomas - https://programmingthomas.wordpress.com/2013/04/03/n-sided-shapes/
-        let sideCount = sideCountNumerator * sideCountDenominator;
-        let decimalSides = sideCountNumerator / sideCountDenominator;
-        let interiorAngleDegrees = (180 * (decimalSides - 2)) / decimalSides;
-        let interiorAngle = Math.PI - Math.PI * interiorAngleDegrees / 180; // convert to radians
+        const sideCount = sideCountNumerator * sideCountDenominator;
+        const decimalSides = sideCountNumerator / sideCountDenominator;
+        const interiorAngleDegrees = (180 * (decimalSides - 2)) / decimalSides;
+        const interiorAngle = Math.PI - Math.PI * interiorAngleDegrees / 180; // convert to radians
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 180](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)